include/superior_mysqlpp/prepared_statements/initialize_bindings.hpp

```diff
@@ -270,8 +355,10 @@ namespace SuperiorMySqlpp
         };
 
 
-        /*
-         * Initialize Bindings
+        /**
```

Comment:
This is probably the only "importantly-visible exported" method from this module. Please describe also in java doc, how does it works.